man/man8/zpool-attach.8

```diff
@@ -62,6 +69,38 @@ creates a three-way mirror, and so on.
 In either case,
 .Ar new_device
 begins to resilver immediately and any running scrub is cancelled.
+.Pp
+If the existing device is a RAIDZ device (e.g. specified as "raidz2-0"), the new
+device will become part of that RAIDZ group.
+A "raidz expansion" will be initiated, and the new device will contribute
+additional space to the RAIDZ group once the expansion completes.
+The expansion entails reading all allocated space from existing disks in the
+RAIDZ group, and rewriting it to the new disks in the RAIDZ group (including the
+newly added
+.Ar device ) .
+Its progress can be monitored with
+.Nm zpool Cm status .
+.Pp
+Data redundancy is maintained during and after the expansion.
+If a disk fails while the expansion is in progress, the expansion pauses until
+the health of the RAIDZ vdev is restored (e.g. by replacing the failed disk
+and waiting for reconstruction to complete).
+Expansion does not change the number of failures that can be tolerated
+without data loss (e.g. a RAIDZ2 is still a RAIDZ2 even after expansion).
+A RAIDZ vdev can be expanded multiple times.
+.Pp
+After the expansion completes, old blocks remain with their old data-to-parity
+ratio (e.g. 5-wide RAIDZ2, has 3 data to 2 parity), but distributed among the
+larger set of disks.
+New blocks will be written with the new data-to-parity ratio (e.g. a 5-wide
+RAIDZ2 which has been expanded once to 6-wide, has 4 data to 2 parity).
+However, the RAIDZ vdev's "assumed parity ratio" does not change, so slightly
```

Comment:
either RAID-Z, just "the vdev's", or just "the RAID-Z assumed parity ratio" (the Qq around a p r should get dropped either way)

Reply:
I'm not seeing a `Qq` around here?

Reply:
By Qq I meant the double quotes (`"`) themselves